docs/infra/website.md

```diff
@@ -220,23 +220,31 @@ Additionally, the `permalink` property is commonly used to give a more special U
 
 #### Example project
 
-See [2018-01-01-teleirc.md](https://github.com/FOSSRIT/fossrit.github.io/blob/67282a55a6330a5c61397eaeac92ef9c025f0432/projects/_posts/2018-01-01-teleirc.md):
+See [2020-03-01-teleirc.md](https://github.com/FOSSRIT/fossrit.github.io/blob/6d11475b205bbb564cb5b284b390b4810bec5d9b/projects/_posts/2020-03-01-teleirc.md):
```

Comment:
Why point to the md blob instead of the actual page?

Reply:
@Nolski @ritjoe The context is for someone who is creating and adding new projects to the website, where they will need the raw Markdown format to create new projects. I presume if someone wants to find an example of a rendered project, they can browse the website itself. :slightly_smiling_face: